src/components/document/component/FieldSubjectCategory.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import {Select, Value} from "baseui/select";
+import {codelist, ListName} from "../../../service/Codelist";
+import {DocumentInformationTypes, DocumentTableRow} from "../../../constants";
+import index from "../../Purpose/Accordion";
+import {FieldArrayRenderProps} from "formik";
+
+const FieldSubjectCategory = (props: {
+  value?: string,
+  rowData:DocumentTableRow,
+  setRowData:Function,
+  index:number,
+  arrayHelpers:FieldArrayRenderProps}) => {
+
+  const [value, setValue] = React.useState<Value>(props.value ? [{
+    id: props.value,
+    label: codelist.getShortname(ListName.SUBJECT_CATEGORY, props.value)
+  }] : []);
+
+  return (
+    <Select
+      options={codelist.getParsedOptions(ListName.SUBJECT_CATEGORY)}
+      onChange={({value}) => {
+        setValue(value);
+        let newRowData = props.rowData;
```

Comment:
bruk {...props.rowData} i det minste for å unngå redigering av objekt i linja under.